packages/app/src/cli/services/dev/app-events/file-watcher.ts

```diff
@@ -50,18 +50,14 @@ export interface OutputContextOptions {
 export class FileWatcher {
   private currentEvents: WatcherEvent[] = []
   private extensionPaths: string[] = []
-  private app: AppLinkedInterface
+  private app: AppInterface
   private readonly options: OutputContextOptions
   private onChangeCallback?: (events: WatcherEvent[]) => void
   private watcher?: FSWatcher
   private readonly debouncedEmit: () => void
   private readonly ignored: {[key: string]: ignore.Ignore | undefined} = {}
 
-  constructor(
-    app: AppLinkedInterface,
-    options: OutputContextOptions,
-    debounceTime: number = DEFAULT_DEBOUNCE_TIME_IN_MS,
-  ) {
+  constructor(app: AppInterface, options: OutputContextOptions, debounceTime: number = DEFAULT_DEBOUNCE_TIME_IN_MS) {
```

Comment:
why the changes to this file? the `function build` command doesn't use the FileWatcher no?